### PR TITLE
Sanitize user submitted content

### PR DESCRIPTION
**Pull request by mhaddy**
Utilize  appendUntrusted() to safely escape any content from the user (e.g., malicious HTML or javascript).

### CODE REVIEW

File: google-apps-script.js [hunk 1]
```diff
@@ -13,13 +13,23 @@ function formatMailBody(obj, order) {
...
     // for every key, concatenate an `<h4 />`/`<div />` pairing of the key name and its value, 
     // and append it to the `result` string created at the start.
   }
   return result; // once the looping is done, `result` will be one long string to put in the email body
 }
 
+// sanitize content from the user - trust no one 
+// ref: https://developers.google.com/apps-script/reference/html/html-output#appendUntrusted(String)
+function cleanMe(theunclean) {
+   var output = HtmlService.createHtmlOutput(theunclean);
```
Comment (mckennapsean) [inline google-apps-script.js]: this line is not used.

File: google-apps-script.js [hunk 2]
```diff
@@ -13,13 +13,23 @@ function formatMailBody(obj, order) {
...
+    result += "<h4 style='text-transform: capitalize; margin-bottom: 0'>" + key + "</h4><div>" + cleanMe(obj[key]) + "</div>";
     // for every key, concatenate an `<h4 />`/`<div />` pairing of the key name and its value, 
     // and append it to the `result` string created at the start.
   }
   return result; // once the looping is done, `result` will be one long string to put in the email body
 }
 
+// sanitize content from the user - trust no one 
+// ref: https://developers.google.com/apps-script/reference/html/html-output#appendUntrusted(String)
+function cleanMe(theunclean) {
```
Comment (mckennapsean) [inline google-apps-script.js]: I agree with @nelsonic, let's try and clean up the names here. e.g. `sanitizeFormResult(rawResult)`, or something along those lines

File: google-apps-script.js [hunk 3]
```diff
@@ -13,13 +13,23 @@ function formatMailBody(obj, order) {
...
+   var output = HtmlService.createHtmlOutput(theunclean);
+   var placeholder = HtmlService.createHtmlOutput(' ');
```
Comment (mckennapsean) [inline google-apps-script.js]: I would use an empty string here to "save space", granted that is probably negligible. Also, code convention in the repo is to use double quotes, i.e. `""`

**Comment by mhaddy** [comment]
@nelsonic @mckennapsean Please review the changes - I believe I've addressed all of your feedback.

**Comment by mckennapsean** [comment]
Thanks for cleaning that up! Will get it merged soon.

**Comment by mckennapsean** [comment]
Done, deployed in production.

**Pull request merged**